File: prac/multipop.c
#include <stdio.h>
int stack[7];
int top=-1;
void push(int ele){
    if(top>=6){
        printf("isfull\n");
    }
    else{
        stack[++top]=ele;
    }
}
int pop()
{
    if (top==-1){
        printf("isEmpty\n");
    }
    else{
        return stack[top--];
    }
    
}
void multipop(int n){
    int j;
    if (n<top+1){
        j=n;
    }
    else{
        j=top+1;
    }
    for(int i=0;i<j;i++){
        printf("%d\n",pop());
    }
}
int main()
{
    push(1);
    push(2);
        push(3);
    push(4);
        push(5);
    push(6);
        push(7);
    push(8);

    multipop(5);
    for(int i=0;i<=top;i++){
        printf("%d ",stack[i]);
    }
    return 0;
}

File: multipop.py
stack=[]
def push(ele):
    stack.append(ele)

def pop():
    if len(stack)==0:
        print("underflow")
    else:
        print(stack.pop())

def multipop(k):
    for i in range(0,k):
        pop()

choice=1
while True:
    if choice==1:
        print("enter element to push : ")
        ele=int(input())
        push(ele)
        choice=-1
    elif choice==2:
        pop()
        choice=-1  
    elif choice==3:
        print("enter k : ")
        k=int(input())
        multipop(k)
        choice=-1
    elif choice==4:
        print(stack)
        choice=-1
    elif choice==0:
        break
    elif choice==-1:
        print("***********0==exit\t 1==push\t 2==pop\t 3==multipop\t 4==print stack**********")
        choice=int(input("enter choice : "))

    
